standalone-metastore/src/main/java/org/apache/hadoop/hive/metastore/HiveAlterHandler.java

```diff
@@ -252,6 +253,13 @@ public void alterTable(RawStore msdb, Warehouse wh, String catName, String dbnam
                 " failed to move data due to: '" + getSimpleMessage(e)
                 + "' See hive log file for details.");
           }
+
+          if (!HiveMetaStore.isRenameAllowed(olddb, db)) {
+            LOG.error("Alter Table operation for " + dbname + "." + name + " failed");
```

Comment:
Shall log both old and new names. Also, error log can be changed to say "Alter table to rename table across database is not allowed for REPL enabled database."